### ISSUE TITLE
tmap_leaflet add.titles option is not displaying title for interactive map in "view" mode

### ISSUE THREAD

**Issue by samantaraymadhu6** [mtennekes/tmap#520]
```
mapRace <- tm_shape(block_spt) +
  tm_polygons("Rc_Prcn", palette = "YlGnBu",id="GEOID",title = "Percentage")+
  tm_facets(by = "racetyp", nrow = 3,free.coords = FALSE)+
  tm_layout(title = "2010 Covington Race Distribution")
```

If I run mapRace with tmap_mode("plot") then the title comes properly but when mode is changed to 'view' for interactive map, the title is not coming.

tmap_leaflet(mapRace, mode = "view", show = TRUE, add.titles = TRUE) is also not giving the title. Is there a way to fix this in interactive map.  Looking for some help.

**Comment by mtennekes** [comment]
Can you provide a reproducible example, please?

**Comment by mtennekes** [comment]
Fixed, see #550
